samples/react-command-print/src/extensions/print/components/print-dialog-content/print-dialog-content.tsx

```diff
@@ -58,14 +58,28 @@ export default class PrintDialogContent extends React.Component<IPrintDialogCont
         initializeIcons();
     }
 
-    public componentDidMount() {
+    public async componentDidMount() {
 
         // Validate and create Print Settings list --> added list definition to elements.xml for adding Print List Settings
         // this.initializeSettings();
         // Get templates
-        this.getTemplates();
+        //this.getTemplates();
         // Get select item values
-        this.getItemContent();
+        //this.getItemContent();
```

Comment:
is it actually ok we are commenting this out? won't we like change the behavior of the sample?

Reply:
I enhanced it by using, since the original sample version was giving showing spinner continously, so i had to adjust the states too

```
await Promise.all([this.getTemplates(), this.getItemContent()]);
```

Instead calling the methods as below, 

```
    public componentDidMount() {

        // Validate and create Print Settings list --> added list definition to elements.xml for adding Print List Settings
        // this.initializeSettings();
        // Get templates
        this.getTemplates();
        // Get select item values
        this.getItemContent();
    }
```

Reply:
ok then we should remove this commented out code. If we improved it and confirmed that it works and it's better I see little sense in leaving commented out code in sample. Fro someone new to the sample it will just create more confusion and question why those comments are here and why there is alternative code but commented out